R/tufte_book.R

```diff
@@ -0,0 +1,86 @@
+
+#' Tufte book format (PDF)
+#' 
+#' Template for creating a book according to the style of 
+#' Edward R. Tufte and Richard Feynman. 
+#' 
+#' @inheritParams pdf_document
+#' 
+#' @export
+tufte_book <- function(fig_width = 4,
```

Comment:
Let's use the same order of parameters as used for pdf_document (e.g. toc options are first, pandoc_args are at the end, etc.).